extensions/sql-database-projects/src/common/azureFunctionsUtils.ts

```diff
@@ -41,6 +41,36 @@ export async function getLocalSettingsJson(localSettingsPath: string): Promise<I
 	};
 }
 
+/**
+ * Adds a new setting to a project's local.settings.json file
+ * modified from setLocalAppSetting code from vscode-azurefunctions extension
```

Comment:
Why did we have to modify this code? What's different about it?

Reply:
the AF extension version has an additional parameter to ignore prompting when the setting already exists and not change the existing setting, but I got rid of it so that it always prompts the user if they want to overwrite. There are a couple other small changes like calling vscode.window.showWarningMessage directly instead of using the wrapper in the AF extension.